modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/dto/Hyperparams.java

```diff
@@ -24,9 +24,8 @@
  */
 
 @Data
-public class ModelParams {
+public class Hyperparams {
 
-    private String modelUUID;
     private String metricKey;
```

Comment:
Why are we binding to a specific metric here? Does the hyperparams object need to know about the metric at all? In fact does it need to know about the model?

I wonder--do we need a Hyperparams class at all? Won't it simply store a dictionary that maps a hyperparam to a hyperparam value? If so, could we just use a Properties in the model itself? (I.e. what would Hyperparams class be doing other than wrapping a Properties?)